sdk/eventgrid/Microsoft.Azure.EventGrid/src/Customization/EventTypes.cs

```diff
@@ -51,7 +63,7 @@ public class EventTypes
         public const string MediaLiveEventIncomingStreamReceivedEvent = "Microsoft.Media.LiveEventIncomingStreamReceived";
         public const string MediaLiveEventIncomingStreamsOutOfSyncEvent = "Microsoft.Media.LiveEventIncomingStreamsOutOfSync";
         public const string MediaLiveEventIncomingVideoStreamsOutOfSyncEvent = "Microsoft.Media.LiveEventIncomingVideoStreamsOutOfSync";
-        public const string MediaLiveEventIncomingChunkDroppedEvent = "Microsoft.Media.LiveEventIncomingChunkDropped";
+        public const string MediaLiveEventIncomingChunkDroppedEvent = "Microsoft.Media.LiveEventIncomingDataChunkDropped";
```

Comment:
Would removing this be a breaking change?

Reply:
Hi Alex, good question. No, this won't be a breaking change. The reason is that the old name of the event was incorrect and it would have never worked since the Media Services publishes the event as LiveEventIncomingDataChunkDropped. I found this issue while updating the corresponding Java SDK and confirmed with the Media Services team that the new name above is the correct name to use.